symengine/tests/test_subs.py

```diff
@@ -1,3 +1,5 @@
+import numpy as np
```

Comment:
This would make the tests fail when numpy is not installed. Can you move this to the function where it's needed and skip it if numpy is installed? See https://github.com/symengine/symengine.py/blob/04ae3b128185faca98bf966cf2a1897250cb31b2/symengine/tests/test_pickling.py#L6

Reply:
Done in: https://github.com/symengine/symengine.py/pull/352/commits/65ceced4a5f0a80cb1fe15b7ad09e056cf3c0b7e